### PR TITLE
zh-translation:/content/en/docs/reference/glossary/service-endpoint.md

### PR DESCRIPTION
**Pull request by JuwanXu**
ref: https://github.com/servicemesher/istio-official-translation/issues/1097

### CODE REVIEW

File: content/en/docs/reference/glossary/service-endpoint.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
 ---
 title: Service Endpoint
 ---
-The network-reachable manifestation of a [service](#service).
-[Workload instances](#workload-instance) expose service endpoints but not all
-services have service endpoints.
+Service Endpoint 是一个 [service](#service) 的网络可达表现形式。Service endpoint 由 [workload instances](#workload-instance) 暴露，并不是所有的服务都有 service endpoint。
```
Comment (rootsongjc) [inline content/en/docs/reference/glossary/service-endpoint.md]: You have also changed the English docs?

Comment (JuwanXu) [reply content/en/docs/reference/glossary/service-endpoint.md]: 改错文件了，我重新提吧

**Pull request closed without merging**